nbs/methods.ipynb

```diff
@@ -44,7 +44,7 @@
     "from typing import Callable, Dict, List, Optional, Union\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    from numba import njit, prange
</pre><p>The prange isn't used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>